### PR TITLE
Add $NIM_pool as a public staking pool

### PR DESCRIPTION
**Pull request by gallareton**

### CODE REVIEW

**Comment by Albermonte** [comment]
The fee here (5%) and the fee in your web is different (2% + 6.9%)

File: public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "$NIM_pool",
+  "description": "Community driven staking pool FROM the people FOR the people!",
+  "website": "https://pool.gallareton.pl/",
+  "address": "NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M",
+  "fee": 0.05,
+  "payoutType": "direct",
+  "payoutSchedule": "every 8 hour",
+  "contact": {
+    "email": "snimpool@proton.me",
+    "telegram": "https://t.me/+sQ6pVI_-ee9jZmJk",
```
Comment (onmax) [inline public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json]: Please your telegram username like `@nimiq`

File: public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "twitter": "https://x.com/MpoolNi49497"
```
Comment (onmax) [inline public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json:12]: Please use username like `@nimiq`

**Review by onmax — CHANGES_REQUESTED** [review]
Thanks for your PR!

File: public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json (hunk 1 above)
Comment (konieczkocoin) [reply public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json]: Can You pleas elaborate? this is a link to the channel
I understand that I have to give my nickname on Telegram?
its: Koniezcko

**Comment by gallareton** [comment]
> The fee here (5%) and the fee in your web is different (2% + 6.9%)

The site contains placeholders for now but nevertheless we fixed the values displayed on site and added a simple explanation for boxes (this "2%" is not a fee).

File: public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json (hunk 1 above)
Comment (onmax) [reply public/validators/albatross/NQ29 FBVT B4GM S27H UBP4 1MTC GNKQ VPBT 099M.json]: You can find the username going to settings in your telegram account.

Then you can use `https://t.me/{USERNAME}` to test it. In my case is https://t.me/onmax_web but I cannot find your handle: https://t.me/Koniezcko.

image

**Comment by onmax** [comment]
Thank you!

**Pull request merged**